packages/flutter/lib/src/material/tabs.dart

```diff
@@ -692,6 +693,13 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// the tab widget itself.
   final Decoration? indicator;
 
+  /// Whether this tab bar should automatically adjust the [indicatorColor].
+  ///
+  /// If [automaticIndicatorColorAdjustment] is true,
+  /// then the [indicatorColor] will be automatically adjusted to [Colors.white]
+  /// when the [indicatorColor] is same as [Material.color].
```

Comment:
It might be worth being specific about which `Material.color` you're referencing here:
```suggestion
  /// when the [indicatorColor] is same as [Material.color] of the [Material] parent widget.
```

Reply:
Done.